docs/pages/docs/components/button.mdx

```diff
@@ -92,3 +93,142 @@ These sizes are standardized across form components.
 </Container>
 `}
 />
+## types
+
+<Playground
+  code={`
+<Container row direction={['column', 'row', 'row']} gap={10}>
+  <Container align="center" justify={"center"}>
+    <Button type={"primary"}>Primary</Button>
+  </Container>
+
+  <Container align="center" justify={"center"}>
+        <Button type={"secondary"}>Secondary</Button>
+  </Container>
+
+  <Container align="center" justify={"center"}>
+   <Button type={"danger"}>Danger</Button>
+  </Container>
+  <Container align="center" justify={"center"}>
+     <Button type={"warning"}>Warning</Button>
+  </Container>
+  <Container align="center" justify={"center"}>
+  <Button type={"success"}>Success</Button>
+  </Container>
+  <Container align="center" justify={"center"}>
+        <Button type={"info"}>Info</Button>
+  </Container>
+  <Container align="center" justify={"center"}>
+
+          <Button type={"dark"}>Dark</Button>
+  </Container>
+  <Container align="center" justify={"center"}>
+      <Button type={"light"}>Light</Button>
+  </Container>
+</Container>
+`}
+/>
+
+## variant-ghost
```

Comment:
It's a title so it must be capitalized, "variant-ghost" ➡️ "Variant - Ghost"

Reply:
Changes has been done 
Please review it

Reply:
Hi @maddy020 
Thank you for contributing to our project :)
When you make the requested changes, please resolve the conversations
And if you want a reviewer to review your code again, click on the "re-request review" button (the one with the arrows)
![image](https://user-images.githubusercontent.com/59099468/206897919-b390ca9c-0f10-4740-9526-e0ca8357536c.png)

Reply:
Sure 
I have requested for re-review